### DISCUSSION TITLE
Find On Page and Fuzzy Search

### DISCUSSION THREAD

**Discussion by some1awesome** [stephanrauh/ngx-extended-pdf-viewer#1963] [Q&A]
I've updated my app to use version 18 and lost the find on page and fuzzySearch capabilities.  Is this something that is gone forever?  Or are we going to get some of this capability back in the future?

**Comment by stephanrauh** [comment]
Yes, these two features are a thing of the past. I'm unhappy about it, but I couldn't continue maintaing my modification of the search engine because of a major merge conflict with pdf.js. The base library added a lot of features to the search bar, and I couldn't add them to ngx-extended-pdf-viewer because of the fuzzy search. After delaying the decision for many months, I reluctanty decided to remove these features.
